pages/player/[uuid]/index.tsx

```diff
@@ -48,15 +46,8 @@ function PlayerDetails(props: Props) {
         if (!isClientSideRendering()) {
             return
         }
-        api.getPlayerName(uuid).then(name => {
```

Comment:
We still need this refresh of the playername.
If the uuid changes through a shallow navigation (we are still on the same page, but the parameter in the url changed), the page is not rendered server side again, therefore props.player still includes the wrong one.

While this may lead to one unnecessary request, it is worth it:

- the benefits of having the player loaded on the server side first and then in the server side rendered page are worth it
- searching the name for a uuid is a very cheap request
- the request is still  cached, as it was performed during the SSR

Reply:
I'm having some trouble reproducing this bug on the localhost development server.
The steps I took was searching for `http://localhost:3000/player/00462df2aabf4f9e8673604970dc8195` before swapping to `http://localhost:3000/player/0a683fa6e4ef4cd1bc0d0754ffeccaa9`. 

From the results that I observed, props.player.uuid had updated. As a result, useEffect was properly called in index.tsx and the selectedPlayer was properly updated.

Reply:
Yeah, seems like it does update the props...
Never mind then